app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :lockable
```

Comment:
```suggestion
  devise :database_authenticatable, :rememberable, :validatable,
         :lockable
```
Giới hạn các tính năng cho đúng